src/LanguageServer.ts

```diff
@@ -1272,9 +1273,11 @@ export class LanguageServer {
             //find the first program that has this file, since it would be incredibly inefficient to generate semantic tokens for the same file multiple times.
             if (project.builder.program.hasFile(srcPath)) {
                 let semanticTokens = project.builder.program.getSemanticTokens(srcPath);
-                return {
-                    data: encodeSemanticTokens(semanticTokens)
-                } as SemanticTokens;
+                if (semanticTokens !== undefined) {
+                    return {
+                        data: encodeSemanticTokens(semanticTokens)
+                    } as SemanticTokens;
+                }
```

Comment:
I've confirmed that is `encodeSemanticTokens` is called with `undefined` that an error will be thrown. So this is another behavioral change, which causes the algorithm to silently continue rather than throw an error in an edge case which I'm not sure will ever happen.